components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1001,7 +1018,7 @@ void RewardsServiceImpl::TriggerOnGrant(ledger::Result result,
 }
 
 void RewardsServiceImpl::GetGrantCaptcha() {
-  ledger_->GetGrantCaptcha();
+  ledger_->GetGrantCaptcha(is_android_);
```

Comment:
can you link me to the ledger changes for this? Passing an `is_android` flag seems a bit hacky to me